files/routes/__init__.py

```diff
@@ -14,4 +14,5 @@
 from .feeds import *
 from .awards import *
 from .giphy import *
+from .dev import *
```

Comment:
Any way to have stronger protections to keep these routes out of production? Conditional import? Or a `@debug_only` decorator just before the routing decorator on all the routes?

Reply:
need to test if this causes import weirdness (no importmania on this fork 😔) but would prefer the conditional import if at all possible tbh